resources/migrations/001_update_migrations.yaml

```diff
@@ -9733,6 +9733,20 @@ databaseChangeLog:
         - customChange:
             class: "metabase.db.custom_migrations.CreateSystemNotificationUserJoined"
 
+  - changeSet:
+      id: v52.2024-10-01T08:00:02
+      author: qnkhuat
+      comment: Migrate user invited email to system event notification
+      rollback: # not necessary, will be removed with the table
+      preConditions:
+        - onFail: MARK_RAN
+        - sqlCheck:
+            expectedResult: 0
+            sql: SELECT count(*) FROM notification_subscription WHERE type = 'notification-subscription/system-event' AND event_name = 'event/alert-create'
+      changes:
+        - customChange:
+            class: "metabase.db.custom_migrations.CreateSystemNotificationAlertCreated"
```

Comment:
To be honest, adding every notification as a migration doesn't feel very appealing to me. Let's discuss our options off-github somewhere?

Reply:
sure